adoc/retail_chap_install.adoc

```diff
@@ -9,7 +9,8 @@ include::entities.adoc[]
 [[retail.sect.running.requirements]]
 == Requirements
 
-Before you install {smr}, ensure your environment meets the minimum requirements:
+Before you install {smr}, ensure your environment meets the minimum requirements.
+In addition to the {susemgr} requirements listed at https://www.suse.com/documentation/suse-manager-3/3.2/susemanager-getting-started/single-html/book.suma.getting-started/book.suma.getting-started.html#quickstart.sect.prereq.hardware and https://www.suse.com/documentation/suse-manager-3/singlehtml/suse_manager21/book_susemanager_proxyquick/book_susemanager_proxyquick.html your {smr} installation will require:
```

Comment:
It feels wrong referencing the **2.1** proxy QS.

Reply:
Huh, good spot. Might need to check with @jsrain

Reply:
@Loquacity Seems that Proxy does not have a standalone book in 3.2, but is only covered by the SUSE Manager Advanced Topics Guide. The link below should provide the needed requirements:

https://www.suse.com/documentation/suse-manager-3/3.2/susemanager-advanced-topics/single-html/book.suma.advanced.topics/book.suma.advanced.topics.html#at.manager.proxy.requirements

Reply:
https://github.com/SUSE/doc-susemanager/pull/209

Also, @keichwa apparently we've been doing IDs and xrefs all wrong :S https://github.com/SUSE/doc-susemanager/wiki/asciidoc-getting-started#links